File: Desenvolvimento/client/src/@fuse/types/models/processo-autor.ts
import { Processo } from './processo';
import { Autor } from './autor';
import { BaseModel } from '../base-model';

export class ProcessoAutor extends BaseModel {
    constructor(
        public id: number,
        public processoId?: number,
        public autorId?: number,
        public autor?: Autor,
        public processo?: Processo,
    ) { super(); }
}

File: Desenvolvimento/server/src/ApplicationCore/Entities/ProcessoAutor.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class ProcessoAutor : IBaseEntity<long>
    {
        public long Id { get; set; }
        public long ProcessoId { get; set; }
        public long AutorId { get; set; }
        public Processo Processo { get; set; }
        public Autor Autor { get; set; }
    }
}
